docs/release-notes/whats-new-3.rst

```diff
@@ -0,0 +1,64 @@
+.. py:currentmodule:: litestar
+
+
+What's changed in 3.0?
+======================
+
+This document is an overview of the changes between version **2.x** and **3.0**.
+For a detailed list of all changes, including changes between versions leading up to the
+3.0 release, consult the :doc:`/release-notes/changelog`.
+
+
+Imports
+-------
+
++----------------------------------------------------+------------------------------------------------------------------------+
+| ``2.x``                                            | ``3.x``                                                                |
++====================================================+========================================================================+
+| **OpenAPI**                                                                                                                 |
++----------------------------------------------------+------------------------------------------------------------------------+
+| ``litestar.openapi.OpenAPIController``             | :mod:`.openapi.plugins`                                                |
```

Comment:
Not sure if this is really an appropriate import redirection, but I mainly just wanted to keep the structure of the table in place so its there to fill out for other things.